library/src/main/scala/scala/scalajs/js/RegExp.scala

```diff
@@ -117,5 +117,6 @@ object RegExp extends js.Object {
   trait ExecResult extends js.Array[js.UndefOr[String]] {
     var index: Int
     var input: String
+    var groups: js.UndefOr[js.Dictionary[String]]
```

Comment:
A named group that does not capture anything will be present but its value will be undefined:
```js
C:\Users\sjrdo>node
Welcome to Node.js v14.15.0.
Type ".help" for more information.
> var r = new RegExp("foo(?<prop>bar)?baz")
undefined
> r.exec("foobarbaz")
[
  'foobarbaz',
  'bar',
  index: 0,
  input: 'foobarbaz',
  groups: [Object: null prototype] { prop: 'bar' }
]
> r.exec("foobaz")
[
  'foobaz',
  undefined,
  index: 0,
  input: 'foobaz',
  groups: [Object: null prototype] { prop: undefined }
]
```
therefore, the typing should be:
```suggestion
    var groups: js.UndefOr[js.Dictionary[js.UndefOr[String]]]
```

Reply:
Moreover, since this property was added in ES 2018, it should receive a proper Scaladoc comment with an ES 2018 badge. See for example
https://github.com/scala-js/scala-js/blob/814d4d99cecaf351cf22ed365efb5d35c1f59ee2/library/src/main/scala/scala/scalajs/js/BigInt.scala#L18
You'll also need to add the CSS class in
https://github.com/scala-js/scala-js/blob/814d4d99cecaf351cf22ed365efb5d35c1f59ee2/assets/additional-doc-styles.css#L1

Reply:
Fixed